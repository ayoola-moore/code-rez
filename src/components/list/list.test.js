import * as React from 'react' // Flow recommended this
import List from './'

describe('Should Find List component', () => {
    it('should find List', () => {
        const component = shallow(<List />)
        expect(component).toMatchSnapshot()
    })
})
