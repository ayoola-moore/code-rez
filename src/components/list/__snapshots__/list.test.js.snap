// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Find List component should find List 1`] = `
<ul>
  <li />
</ul>
`;
